src/EnergyPlus/WindowManager.hh

```diff
@@ -75,9 +74,16 @@ namespace Window {
 
     int constexpr maxGlassLayers = 5;
     int constexpr maxGapLayers = 5;
-    int constexpr maxIncidentAngles = 20;
```

Comment:
This is never 20, number of elements always hardwired to 10 during use. So why allocate all the arrays to 20?